lithops/standalone/proxy.py

```diff
@@ -60,12 +93,11 @@ def init_keeper():
     config_file = os.path.join(REMOTE_INSTALL_DIR, 'config')
     with open(config_file, 'r') as cf:
         serverfull_config = json.load(cf)
-    handler = StandaloneHandler(serverfull_config)
 
-    if handler.auto_dismantle:
-        keeper = threading.Thread(target=budget_keeper, args=(handler,))
-        keeper.daemon = True
-        keeper.start()
+    handler = StandaloneHandler(serverfull_config)
+    keeper = threading.Thread(target=budget_keeper, args=(handler,))
+    keeper.daemon = True
```

Comment:
I'm not sure if we want to set this thread to be Daemon. Wouldn't we prefer that it will continue to run and stop the VM even in case the main thread crashed?

Reply:
The proxy is installed as a Unix service. If the main thread crashes, the operating system will automatically restart the proxy, so it must be True